### ISSUE TITLE
maximum call stack size exceeded

### ISSUE THREAD

**Issue by bryanlarsen** [jakejs/jake#327]
node: 0.10.48
npm: 3.10.9
jake: 8.0.15

with a file list of 165 files I get

```
(node) warning: Recursive process.nextTick detected. This will break in the next version of node. Please use setImmediate for recursive deferral.
(node) warning: Recursive process.nextTick detected. This will break in the next version of node. Please use setImmediate for recursive deferral.
< repeats very many times >

util.js:35
  var str = String(f).replace(formatRegExp, function(x) {
                      ^
RangeError: Maximum call stack size exceeded
```

If I do `for f in `cat files.txt` ; do echo $f ; ./node_modules/.bin/jake $f ; done`

everything works fine.

**Comment by bryanlarsen** [comment]
replacing `process.nextTick` with `setImmediate` in lib/task/task.js fixes the problem.

I believe this is also PR #280, although I did the change manually.

**Comment by welearnednothing** [comment]
Thanks for reporting this, Bryan. For completeness, can you provide a code sample that I can use to replicate this? Thanks!

**Comment by mde** [comment]
Merged fix from PR #280. Thanks for reporting this!
